src/drivers/intel/cavs/hda-dma.c

```diff
@@ -175,6 +175,8 @@ struct dma_pdata {
 };
 
 static int hda_dma_stop(struct dma *dma, int channel);
+static int hda_dma_avail_data_size(struct hda_chan_data *chan);
```

Comment:
where is the definition of this function? I can't see it anywhere

Reply:
Line 843.

Reply:
Ahh, so it was there before you have just added declaration for it.